cterasdk/core/cloudfs.py

```diff
@@ -43,19 +43,23 @@ def list_folder_groups(self, include=None, user=None):
         param = builder.build()
         return query.iterator(self._portal, '/foldersGroups', param)
 
-    def mkfg(self, name, user=None):
+    def mkfg(self, name, user=None, deduplication_method_type=None, fixed_block_size_kb=None):
         """
         Create a new Folder Group
 
         :param str name: Name of the new folder group
         :param cterasdk.core.types.UserAccount user:
          User account, the user directory and name of the new folder group owner (default to None)
+        :param str deduplication_method_type: Deduplication-Method = <"FixedBlockSize"> / <"averageBlockSizeKb">
```

Comment:
deduplication_method_type should be an enumeration. See core/types.py for examples.

Reply:
Done
i have sent a new Pull Request that follows your guidelines and passed the CI tests.